analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/ProgramStateTest.cs

```diff
@@ -241,9 +247,15 @@ public void ToString_WithCaptures()
         [TestMethod]
         public void ToString_Exception()
         {
-            var sut = ProgramState.Empty.SetException(ExceptionState.UnknownException);
+            var sut = ProgramState.Empty.PushException(ExceptionState.UnknownException);
+            sut.ToString().Should().BeIgnoringLineEndings(
+@"Exception: Unknown
+");
+
+            sut = sut.PushException(ExceptionState.UnknownException);   // There are two exceptions in the stack now
```

Comment:
It would be great to have different type of exceptions to be able to check also the order.

Reply:
That is in `PopException_RemovesInCorrectOrder` test.